returnn/tf/util/basic.py

```diff
@@ -1213,6 +1213,8 @@ def get_activation_function(s):
     return getattr(tf.nn, s)  # e.g. relu, elu, sigmoid, softmax, ...
   elif hasattr(tf, s):
     return getattr(tf, s)  # e.g. log, abs
+  elif hasattr(tf.math, s):
```

Comment:
`tf.math` does not exist in earlier TF versions, I think.
```suggestion
  elif hasattr(tf, "math") and hasattr(tf.math, s):
```

Reply:
It exists at least in tf1.15, but this way we're on the safe side for ealier version.

Reply:
There are couple of people who use TF 1.8 or maybe even earlier.